pkg/scaffold/helm/chart.go

```diff
@@ -265,3 +270,116 @@ func createChartFromRemote(destDir string, opts CreateChartOptions) (*chart.Char
 
 	return createChartFromDisk(destDir, chartArchive, false)
 }
+
+func GenerateRoleRules(chart *chart.Chart) ([]rbacv1.PolicyRule, []error) {
+	helmOperatorRules := []rbacv1.PolicyRule{
+		// We need this rule so tiller can read namespaces to ensure they exist
+		{
```

Comment:
That's a good idea. Unfortunately, the comments get stomped out by [`scaffold.UpdateRoleForResource`](https://github.com/operator-framework/operator-sdk/blob/master/internal/pkg/scaffold/role.go#L49) because of the marshaling and unmarshaling that happens.

Kubernetes has an open issue addressing this problem: https://github.com/kubernetes/kubernetes/issues/15157